packages/components/src/tag/docs/TagGroup.stories.tsx

```diff
@@ -0,0 +1,377 @@
+            </TagList>
+            <HelperMessage>The planets in this list are in no particular order.</HelperMessage>
+        </TagGroup>
+    ),
+    args: {
+        onRemove: (ids: Selection) => {
+            alert(`Remove: ${[...ids]}`);
+        }
+    }
+} satisfies Story;
+
+/**
+ * Tags can be selected using the `selectionMode` prop.
+ * Use `defaultSelectedKeys` for initial selected items (uncontrolled) and `selectedKeys` to manage selected items (controlled). 
+ * The selected keys should match the item's id prop.
+ */
+export const SelectableTags = {
+    ...Description,
+    args: {
+        ...Description.args,
+        selectionMode: "multiple",
+        defaultSelectedKeys: new Set(["1", "3"])
+    }
+} satisfies Story;
+
+/**
+ * Tags can be disabled using the `disabledKeys` prop or the `isDisabled` prop on the Tag component.
+ */
+export const DisabledTags = {
+    ...Description,
+    args: {
+        ...Description.args,
+        disabledKeys: new Set(["1", "2", "3"])
+    }
+} satisfies Story;
+
+/**
+ * If a TagGroup is invalid, it will display an error message. Displaying this error message will hide the helper message.
+ * A individual Tag can also be set as invalid using the `isInvalid` prop.
+ */
+export const Validation = {
+    args: {
+        isInvalid: true,
+        selectionMode: "multiple"
+    },
+    render: function Render(args) {
+        const [isInvalid, setIsInvalid] = useState(args.isInvalid);
+        function onChange(keys: Selection) {
+            // if value is empty, then it is invalid
+            if (typeof keys === "object") {
+                setIsInvalid(keys.size === 0);
+            }
+        }
+
+        return (
+            <Stack>
+                <TagGroup {...args} onSelectionChange={onChange} isInvalid={isInvalid}>
+                    <HelperMessage>Unselect all to show the error message</HelperMessage>
+                    <ErrorMessage>Select a tag and the description will appear</ErrorMessage>
+                    <TagList>
+                        <Tag id="1">Tag 1</Tag>
+                        <Tag id="2">Tag 2</Tag>
+                        <Tag id="3">Tag 3</Tag>
+                    </TagList>
+                    <Label>List of Tags</Label>
+                </TagGroup>
+                <TagGroup onSelectionChange={onChange}>
+                    <Label>List of Tags with only one invalid</Label>
+                    <TagList>
+                        <Tag id="1">Tag 1</Tag>
+                        <Tag id="2" isInvalid>Tag 2</Tag>
+                        <Tag id="3">Tag 3</Tag>
+                    </TagList>
+                </TagGroup>
+            </Stack>
+        );
+    }
+} satisfies Story;
+
+/** 
+ * Customize the empty state message when there are no tags.
+ */
+export const EmptyState = {
+    render: props => (
+        <TagGroup {...props} aria-label="tag-group">
+            <TagList
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/38871812/8468cb16-3a07-49a4-a85d-d03bec05ad22)

There should be a focus style on the TagList